utils/rpms/cart.spec

```diff
@@ -138,6 +138,10 @@ ln %{?buildroot}%{carthome}/{TESTING/.build_vars,.build_vars-Linux}.sh
 %{carthome}/.build_vars-Linux.sh
 
 %changelog
+* Thu Dec 5 2019 Alexander Oganezov <alexander.a.oganezov@intel.com> - 3.2.0-3
+- Libcart version 3.2.0-3
+- Restrict mercury to be < 1.0.1-22
+
 * Tue Dec 3 2019 Alexander Oganezov <alexander.a.oganezov@intel.com> - 3.2.0-2
```

Comment:
Hrm.  Why is this one 3.2.0?  Isn't this PR the first to go to 3.2.0?  Maybe this is just a typo that needs fixing?

Reply:
No we've been at 3.2 for a while, but cart.spec had omission where version was still 3.1 there